src/GitVersion.Core/VersionCalculation/VersionSearchStrategies/BaseVersionOperator.cs

```diff
@@ -6,6 +6,8 @@ public sealed record BaseVersionOperator : IBaseVersionIncrement
 {
     public string Source { get; init; } = string.Empty;
 
+    public VersionIncrementSourceType SourceType { get; } = VersionIncrementSourceType.Tree;
```

Comment:
Can you please explain me for what the enum `VersionIncrementSourceType` is good for? In my opinion the hole SourceType property can be removed.